### PR TITLE
Fish factory

### PR DESCRIPTION
**Pull request by Exclaminator**

### CODE REVIEW

**Comment by MichielvdBerg** [comment]
Looks good, I approve.

**Comment by spiderbiggen** [comment]
Looks good :8ball:

File: src/main/java/lasersharks/enemies/FishFactory.java [hunk 1]
```diff
@@ -34,20 +34,35 @@
...
-  @SuppressWarnings("unchecked")
-  private static final Class<? extends FishBot>[] FISH_CLASSES = new Class[] { Enemy1.class,
-      Enemy2.class, Enemy4.class, Enemy5.class, Enemy6.class, Enemy7.class, Enemy8.class,
-      Enemy10.class, Enemy12.class };
-
-  @SuppressWarnings("unchecked")
-  private static final Class<? extends Object>[] CONSTRUCTOR_HEAD = new Class[] { Position.class,
-      java.lang.Float.class, java.lang.Double.class, Direction.class };
-
+
```
Comment (SytzeAndr) [inline src/main/java/lasersharks/enemies/FishFactory.java]: Is there a simple way to link these two strings together? If not, please put a comment how these two arrays are linked by index.

**Comment by SytzeAndr** [comment]
apart from my comment, looks good. can be merged.

**Pull request merged**